apps/deploy-web/src/components/providers/ProviderTableRow.tsx

```diff
@@ -58,7 +58,7 @@ export const ProviderListRow: React.FunctionComponent<Props> = ({ provider }) =>
 
   return (
     <TableRow className="provider-list-row cursor-pointer hover:bg-muted-foreground/10 [&>td]:px-2 [&>td]:py-1" onClick={onRowClick}>
-      {provider.isOnline ? (
+      {provider.name ? (
```

Comment:
question (blocking): What's the reasoning behind using the instead of `isOnline`?

Reply:
Ah ok yea I wasn't sure, but sure makes sense because this is to deplay the name anyways.